crates/quantum_info/src/pauli_lindblad_map/pauli_lindblad_map_class.rs

```diff
@@ -384,24 +384,56 @@ impl PauliLindbladMap {
     }
 
     /// Sample sign and Pauli operator pairs from the map.
-    pub fn sample(&self, num_samples: u64, seed: Option<u64>) -> (Vec<bool>, QubitSparsePauliList) {
+    /// Note that here the "sign" bool is interpreted as the exponent of (-1)^b, which is the
+    /// opposite convention to the sample function.
```

Comment:
I wasn't sure, should this be changed from "sample function" to "signed_sample function"?
```suggestion
    /// opposite convention to the signed_sample function.
```

Reply:
Actually this is in the rust class and is referencing the rust-class `PauliLindbladMap.sample` method, which no longer exists, there is only `parity_sample` now. I'm going to remove this statement.